specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1028,7 +1029,9 @@ components:
       description: ""
       properties:
         AKRregistercode:
-          $ref: "#/components/schemas/Waardelijst"
+          allOf:
```

Comment:
Dit verschilt met de BAG. Daar hebben we ervoor gekozen dat we de allOf niet hiervoor te gebruiken. In OAS 3.1. is het wel toegestaan om een description als sibling aan een $ref toe te voegen

Reply:
Dat klopt, ik heb hiervoor gekozen om direct de link zichtbaar te maken. Ook in de grafische interface zodat developers daar direct op in kunnen klikken. Als dit een issue voor code-genereren oplevert doen we het anders. Maar dit levert ook geen warning op in 3.0.

Reply:
Graag de BRK & BAG API consistent houden.

Reply:
> 
> 
> Dat klopt, ik heb hiervoor gekozen om direct de link zichtbaar te maken. Ook in de grafische interface zodat developers daar direct op in kunnen klikken. Als dit een issue voor code-genereren oplevert doen we het anders. Maar dit levert ook geen warning op in 3.0.

De allOf constructie zorgt er wel voor dat er een class wordt gegenereerd die de Waardelijst inherit, maar geen extra properties heeft. Ter indicatie, deze yaml levert nu 25 extra classes die allemaal Waardelijst inheriten

Reply:
Duidelijk argument. Ik trek deze gelijk aan hoe we dat in de BAG gedaan hebben.